views/tenants/index.erb

```diff
@@ -1 +1,9 @@
-Listing tenants for apartment #3
+<h2>Tenants of <%= @apartment.address%></h2>
+<ul>
+<%@apartment.tenants.order(name: :desc).each do |tenant|%>
```

Comment:
similar to above comment, consider creating an instance variable in the controller that does the sorting, rather than in the view.